20230301/mike.java

```diff
@@ -0,0 +1,44 @@
+/*
+To practice different algos, I use quicksort to complete this question.
+Althrough quicksort may exceed O(nlogn), it pass all of the test case on LeetCode.
+Maybe we should try heapsort and counting sort next time!
+*/
+
+class Solution {
+    public int[] sortArray(int[] nums) {
+        quickSort(nums, 0, nums.length-1);
```

Comment:
is there any official coding style to be followed in java's world? i am just curious which one is more encouraged, `nums.length - 1` or `nums.length-1`?